OneCommunityGlobal/HGNRest
# Anirudh loginmethod unittest

# Description
Created unit tests for the login controller

## How to test:
1. check into the current branch
2. do `npm install` 
3. do `npm run test logincontroller`
All  8 tests should pass. If you run `npx jest logincontroller`, you may get a console error saying try block failed. That is a test case.

![Screenshot 2024-04-26 at 10 31 26 PM](https://github.com/OneCommunityGlobal/HGNRest/assets/44672833/dffa5b6e-cd96-4b39-b65a-0109d35d150f)
